app/controllers/create_phone_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CreatePhoneController < EditPhoneController
```

Comment:
Humm, this is interesting a couple of things caught me off guard here:

- Create inheriting from Edit controller?
- The phrasing of an entire controller around a single endpoint?

What are the advantages to having it this way instead of having a:

```ruby
class PhoneController

  def create; end

  def update; end

end
```

Reply:
Sure, I'll have a go at merging the controllers

Reply:
also happy for that to be a refactor later if we wanted to rationlise a number of these at once?